starport-core/src/main/scala/com/krux/starport/cli/CleanupNonStarportOptionParser.scala

```diff
@@ -13,6 +13,14 @@ object CleanupNonStarportOptionParser extends Reads {
     head(programName)
     help("help").text("prints this usage text")
 
+    opt[Seq[String]]("excludePrefixes").valueName("<excludePrefixes>")
```

Comment:
Also instead of `Seq` I think make it `unbounded` works better in many cases, other wise the prefixes need to be comma separated, and is weird in many cases.